section1/relection.md

```diff
@@ -2,18 +2,76 @@
+
 2. What are the data types you learned about in this section? In your own words, define each.
 
+- Boolean - a Boolean results in a true or false only. Used in 'for loops' and 'if statements'. This is something that I am planning to research more but I know it will come up. Specific rules regarding Booleans resulting in a true output (unless specified) are:
+
+```'a' < 'p'
+
+3 === '3' /* ignores datatype*/
+3 == '3' // false
+
+'2' > '1'
+```
+
+- Number - includes all **number, decimals, negatives, and fractions(not technically, but it works)**. No commas.
+
+- String - consists of **letters, numbers, and other characters in QUOTES (emojis too).**  
+
 3. How would you log the string `"Hello World!"` to the console?
 
+```
+console.log('Hello World!');
+```
+
 4. What is/are the character(s) you would use to indicate comments in a JavaScript file? What is the purpose of a code comment?
 
+/// This is used for single line comments.
+
+/* This is used for multiline comments. The purpose is to deactivate a portion of code in order to execute it without that portion. Or to leave comments for yourself or another member of your team. Ideally the code explains itself, but if that isn't the case then this is a great option! */
+
 5. In your own words, what is a variable? How would you explain it to a 5 year old?
 
+A variable has a name and a value. We can name our variable anything that we want, but it should explain the value. For example if the value is 'John' then the variable name could be 'name'. When you are declaring a variable, you always need to put var, let, or const before it. Declaring a variable looks like this.
+
+```
+var name = 'John';
+```
+
+The = `=` is the operator. Also, variables can change or vary.
+
 6. Think of a site or app you use frequently. What are three variables that are probably used? Which data type would each of those variables probably hold?
 
+Spotify -
+userName and password - 'string'
+likeSong - Boolean resulting in true
+play - Boolean resulting in true
+pause - Boolean resulting in false
+
 7. In your own words, explain what concatenation is.
 
+Concatention is the joining together of strings using the `+` operator.
+
 8. Think of a site or app you use frequently. Where do you think the developers used concatention?
 
-9. What questions do you still have about the work we've done so far? (not having a question is not an option)
\ No newline at end of file
+Every site that has user accounts will likely use this function. Let's take Amazon for example, when you are not logged in, it says 'Hello', however when you log in, it says 'Hello, Nina'.
+
+9. What questions do you still have about the work we've done so far? (not having a question is not an option)
+
+I didn't quite grasp the sheep pseudocode exercise. I didn't know where to get started. Is there a more specific list of questions to ask yourself when attempting to define a goal and write out the steps in pseudocode?
```

Comment:
🚨I'm unsure where you saw a 🐑 pseudocode, drop the link here and I will see what I can figure out

Reply:
@priyapower it was on one of the homework exercises. The exercise was irrelevant. I just need to practice more pseudo coding, which I will do in the next month. I'll look out for videos as that tends to be the best way for me to process information. Thank you for the Khan Academy suggestion for CSS/html. It's great!

Reply:
https://mod0.turing.io/session3/

"Psuedocode Practice: Counting Sheep"

Reply:
Oh gosh, you would think I would know that since I'm teaching that section tonight 🤦🏾‍♀️, but I was racking my brain in the prework trying to find a 🐑 reference! Hhaha

Some suggested readings:
1. https://blog.usejournal.com/how-to-write-pseudocode-a-beginners-guide-29956242698
2. https://www.geeksforgeeks.org/how-to-write-a-pseudo-code/